.github/workflows/api-e2e-multitenant.yml

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Admin API Multi Tenant E2E Tests + ODS 7
+
+on:
+  push:
+    branches: [main]
+  schedule:
+    - cron: "0 5 * * 1"
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+
+env:
+  JIRA_ACCESS_TOKEN: ${{ secrets.JIRA_ACCESS_TOKEN }}
+  ADMIN_API_VERSION: "1.2"
+  PROJECT_ID: "13401"
+  CYCLE_NAME: "Automation Cycle"
+  TASK_NAME: "API Automation Task"
+  FOLDER_NAME: "API Automation Run"
+  RESULTS_FILE: "test-xml-results"
+
+jobs:
+  run-e2e-multitenant-tests:
+    defaults:
+      run:
+        working-directory: ./Application/EdFi.Ods.AdminApi/
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Copy application folder to docker context
+        run: |
+          mkdir ../../Docker/Application
+          cp -r ../EdFi.Ods.AdminApi ../../Docker/Application
+
+      - name: Copy nuget config to docker context
+        run: cp ../NuGet.Config ../../Docker/Application
+
+      - name: Update certificates
+        run: cp -r ../../eng/test-certs/ssl ../../Docker/Settings
+
+      - name: Run Admin API
+        run: |
+          docker compose \
+            -f '../../Docker/Compose/pgsql/MultiTenant/compose-build-dev-multi-tenant.yml' \
+            --env-file './E2E Tests/gh-action-setup/.automation.env' \
+            up -d
+
+      - name: Verify containers are running
+        run: |
+          chmod +x './E2E Tests/gh-action-setup/inspect.sh'
+          './E2E Tests/gh-action-setup/inspect.sh' adminapi-packaged
```

Comment:
@simpat-adam 
We can change the container name to adminapi on docker file "\Docker\Compose\pgsql\MultiTenant\compose-build-dev-multi-tenant.yml".